docs/content/any/reference/tooling/ide.md

```diff
@@ -2,18 +2,33 @@
 title: IDE Support
 aliases:
     - ../more/dev-tools/ide.html
-description: IDE integrations provide syntax highlighting for Polar files.
+description: IDE integrations for working with Oso policies.
 ---
 
 # IDE Support
 
-Oso’s IDE (Integrated Development Environment) integrations provide syntax highlighting for `.polar` files.
+Oso’s IDE (Integrated Development Environment) integrations provide syntax
+highlighting for `.polar` files. Additionally, our Visual Studio Code extension
+will display diagnostics (errors & warnings) from your Oso policy in-line in
+the editor and in VSCode's **Problems** pane.
 
 ## Supported IDEs
 
 ### [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=osohq.oso)
+
+#### Features
+
+- Syntax highlighting.
+- Diagnostics (errors & warnings) from your Oso policy are displayed in-line in
+  the editor and in VSCode's **Problems** pane.
+
 ### [Vim](https://github.com/osohq/polar.vim)
 
+#### Features
+
+- Syntax highlighting.
+
 ### Want support for your IDE of choice?
 
-Let us know by using our chat on the bottom right to send us a Slack message.
+<!-- TODO(gj): is this still accurate? I don't see the Intercom button on this page. -->
```

Comment:
Highlighting for 👀

Reply:
Yeah your browser is probably just blocking it.

Reply:
```suggestion
Let us know by using our chat on the bottom right to send us a Slack message.
```